ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
I can see this use case is exist, but in current RK framework the `go back` action is not indicating `save my answer`.

If you like to implement this "pick and go" style on iPad, I would suggest you create a custom VC which hosts a list of button/cells on the UI, when user tap any of them, modally present a seperate ORKTaskViewController for user. Each TaskViewController could contain one more question steps.